cmd/package.go

```diff
@@ -272,6 +272,8 @@ func CollectPackage(repo *util.Repo, packageDir string, pullMissing bool, custom
 		}
 	}
 
+	capstanignore := core.CapstanignoreInit(filepath.Join(packageDir, ".capstanignore"), verbose)
```

Comment:
Don't pass `verbose` to `CapstanignoreInit`. I prefer that you do it like this:

1. Check if the `.capstanignore` exists in the package; if it does, pass it to `CapstanignoreInit`, otherwise pass `""` to indicate an empty file
2. In case `CapstanignoreInit` fails to load the file, return an error
3. Handle the error here; in case of `verbose == true`, print the message; stop collecting package